src/components/charts/ChartHeaderFilters.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { ReactComponent as Search } from '../../assets/icons/ic-search.svg';
+import ReactSelect, { components } from 'react-select';
+import { DropdownIndicator, ValueContainer } from './charts.util';
+import { Checkbox, Option, multiSelectStyles, } from '../common';
+import { ReactComponent as Clear } from '../../assets/icons/ic-error.svg';
+import { useRouteMatch, useHistory, useLocation } from 'react-router'
+
+const QueryParams = {
```

Comment:
same constant is defined in more than components

Reply:
fixed